### PR TITLE
feat: filtering by branch

### PR DESCRIPTION
**Pull request by Joshyahweh**

### CODE REVIEW

**Comment by darmhoo** [comment]
What you worked on is not functional. Check how the other filters are being used, inspect the request in your network tab

**Comment by darmhoo** [comment]
the filter name should be branch

**Comment by JoshMTN** [comment]
> What you worked on is not functional. Check how the other filters are being used, inspect the request in your network tab

What do you mean by it's not functional

**Comment by Joshyahweh** [comment]
> What you worked on is not functional. Check how the other filters are being used, inspect the request in your network tab

What do you mean by it's not functional?

**Comment by Joshyahweh** [comment]
> the filter name should be branch

Why should it be branch?

File: src/views/Admin/ReportingTool.vue [hunk 1]
```diff
@@ -418,6 +434,8 @@
 			this.getOrderTypes();
 			this.loaded = true;
 			this.getBarchartColors();
+
+			console.log("Branch", this.branch)
```
Comment (cheedinmar) [inline src/views/Admin/ReportingTool.vue]: Remove console

**Review by cheedinmar — APPROVED** [review]

**Review by darmhoo — APPROVED** [review]

**Pull request merged**